### ISSUE TITLE
Conflict with Chrome native datepicker when using directive

### ISSUE THREAD

**Issue by GuillaumeNury** [vlio20/angular-datepicker#134]
When using dpDayPicker directive like that :
```html
<md-input-container>
    <input mdInput name="fromDate" type="month" [(ngModel)]="fromDate" [dpDayPicker]="fromDatePickerConfig" theme="dp-material" attachTo=".mat-input-wrapper" required="required" />
</md-input-container>
```

```js
this.fromDate = moment();

this.fromDatePickerConfig = {
     format: 'MMM YYYY'
};
```

It works like a charm in Firefox :
![image](https://user-images.githubusercontent.com/12662533/28377726-10f6b256-6cae-11e7-9261-a69959a9d91f.png)

but in Chrome, the native implementation of input[type=month] conflicts with your datepicker :
![image](https://user-images.githubusercontent.com/12662533/28377849-6eac61ca-6cae-11e7-8019-30a2cf3dcc79.png)

In fact, Chrome force the format 'MMMM YYYY', so there is a mismatch.

Could you add an `@Input() calendarMode` in `dpDayPicker` directive, so we could use input[type=text] ?

**Comment by vlio20** [comment]
Ahhhh! I will change the 'type' to 'mode' this will solve your... And it won't conflict with the reserved type word.

**Comment by GuillaumeNury** [comment]
It would be awesome ! 😄

**Comment by vlio20** [comment]
I will try to push it be the end of the weekend (hopefully earlier).

**Comment by vlio20** [comment]
resolved in 		c84e8a6

**Comment by GuillaumeNury** [comment]
I tried version 2.2.0, and I worked like a charm. Thank you !
